pkgs/development/python-modules/maestral/default.nix

```diff
@@ -90,6 +97,19 @@ buildPythonPackage rec {
     "test_locking_multiprocess"
     # OSError: [Errno 95] Operation not supported
     "test_move_preserves_xattrs"
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    # maetral daemon does not start but worked in real environment
+    "test_catching_non_ignored_events"
```

Comment:
Very tiniest of nits: `maetral` -> `maestral`

Reply:
Ahhh, I'll fix it next time. Thanks.